app/styles/radix/navigation-menu.css

```
@@ -0,0 +1,13 @@
+.NavigationMenuRoot {
+  @apply flex flex-row relative justify-center
+  }
+  
+  .NavigationMenuList {
+    @apply flex
+  }
```

saquemos esta indentacion inicial que hay en esta clase 
```suggestion
.NavigationMenuList {
  @apply flex
}
```